CONTRIBUTING.md

```diff
@@ -11,7 +11,7 @@ If you have found a bug in our project or would like to request a new feature, p
 If you'd like to contribute code to our project, please follow these steps:
 
 1. Comment on the issue you'd like to work on (or open a new issue if there isn't one yet) to let us know you're interested in working on it.
-2. Make sure you have `NODE=16.14.0` and `pnpm=8.6` installed on your local machine (or docker/VM) before running the dev server. To know how to run the dev server please refer to `README.md`.
+2. Make sure you have `NODE=20.10.0` (specified in `.node-version`) and `pnpm=10.6.1` (specified in `package.json`) installed on your local machine (or docker/VM) before running the dev server. To know how to run the dev server please refer to `README.md`.
```

Comment:
should we remind that you can use corepack to install correct version of pnpm?

Reply:
no comment on this one, I always install my pnpm from its official site...

Reply:
OK, I think that's not important.

Reply:
thanks for review